packages/client/src/errors.ts

```diff
@@ -0,0 +1,41 @@
+import { AxiosResponse } from 'axios';
+import { ZodError } from 'zod';
+
+interface OPErrorParmeters {
+  errorMessage?: string;
+  errorUri?: string;
+  errorDescription?: string;
```

Comment:
discussed offline, description can be extracted from phrases package  like `sdk-phrases` to align error code/message across SDKs

Reply:
LOG-173